waiter/src/waiter/marathon.clj

```diff
@@ -428,10 +408,10 @@
          (not (str/blank? home-path-prefix))]}
   (when (or (not slave-directory) (not mesos-slave-port))
```

Comment:
Maybe you avoided these renames to avoid breaking configs, but a selective rename is confusing.

Reply:
Undid the selective renames.